tests/php/Integration/Fixtures/Call/global-call.test

```diff
@@ -12,12 +12,12 @@ $GLOBALS["__phel"]["user"]["x"] = new class() extends \Phel\Lang\AbstractFn {
 };
 $GLOBALS["__phel_meta"]["user"]["x"] = \Phel\Lang\TypeFactory::getInstance()->persistentMapFromKVs(
   new \Phel\Lang\Keyword("start-location"), \Phel\Lang\TypeFactory::getInstance()->persistentMapFromKVs(
-    new \Phel\Lang\Keyword("file"), "Call/global-call.test",
+    new \Phel\Lang\Keyword("file"), "string",
```

Comment:
Could we keep the filenames here and in every other test?

Reply:
With the first implementation that I did, it wasn't possible, but I just realize that the second option (the one that it's pushed in this PR) is actually possible and super easy, of course! I will change this 💡